src/config.json

```diff
@@ -1645,5 +1645,6 @@
     "tomocoln.io",
     "docai.science",
     "tomocoin.network"
+    "immvrse.io",
```

Comment:
Bad JSON, cannot add a comma to the end of a list, and not add a comma to another item.